features/vaccineData.js

```diff
@@ -0,0 +1,28 @@
+const vaccine = require("../repositories/vaccineOps.js");
+
+//Should Take in vaccine keyword and two letter state
+module.exports = function (app) {
+  //Regular Expression search (vaccine + State Abbreviation)
+  let vaccineSearch =
+    /vaccine (AL|AK|AZ|AR|CA|CO|CT|DE|FL|GA|HI|ID|IL|IN|IA|KS|KY|LA|ME|D|MA|MI|MN|MS|MO|MT|NE|NV|NH|NJ|NM|NY|NC|ND|OH|OK|OR|PA|RI|SC|SD|TN|TX|UT|VT|VA|WA|WV|WI|WY)/;
+  app.message(vaccineSearch, vaccineData);
+};
+
+async function vaccineData({ message, client }) {
+  //Cut out State Abbriviation into array
+  const stateAbv = message.text.split(" ", 2);
```

Comment:
the name `stateAbv` is a bit misleading here, since this is actually an array, where the second element is the state abbreviation.  we can use destructuring syntax here to only grab the variable we need.
```suggestion
  const [, stateAbv] = message.text.split(" ");
```
since we don't care about the first part of the text (which we know is `vaccine`), we just don't assign a variable to it, hence the first comma.